fdbserver/Resolver.actor.cpp

```diff
@@ -750,6 +749,7 @@ ACTOR Future<Void> resolver(ResolverInterface resolver,
                             InitializeResolverRequest initReq,
                             Reference<AsyncVar<ServerDBInfo> const> db) {
 	try {
+		TraceEvent("ResolverEncryptionAtRestMode").detail("Mode", initReq.encryptMode.toString());
```

Comment:
Any reason we can't update the existing Trace in `resolverCore()`

`TraceEvent("ResolverInit", resolver.id()).detail("RecoveryCount", initReq.recoveryCount);` 

I would rather do that as it allows one to grep for single pattern to learn all relevant details about the resolver?